openmetadata-ui/src/main/resources/ui/playwright/e2e/Pages/Tags.spec.ts

```diff
@@ -154,6 +154,15 @@ test('Classification Page', async ({ page }) => {
       page.locator('[data-testid="add-new-tag-button"]')
     ).toBeDisabled();
 
+    await expect(page.getByTestId('add-domain')).not.toBeVisible();
+    await expect(page.getByTestId('add-owner')).not.toBeVisible();
+
+    await page.getByTestId(tag.responseData.name).click();
```

Comment:
Here we are changing the page, let's add proper api and loader await here, so it won't become flaky